public/data/harshrai654.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Can you move this to a different PR 😅

Reply:
Ok, so should i remove my profile.json file from this PR?

Reply:
Yep